src/teams/team.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Country } from 'src/countries/country.entity';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn
+} from 'typeorm';
+
+@Entity('teams')
+export class Team {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  logo: string;
+
+  @Column()
+  country_id: number;
```

Comment:
This is not needed, by having the country property you defined in the same file line 34, passing the column name and referenced column name at line 33, TypeORM will automatically generate the needed FK